src/test/mina_automation/executor.ml

```diff
@@ -11,83 +11,113 @@ module DockerContext = struct
     { image : string; workdir : string; volume : string; network : string }
 end
 
-type context =
-  | Dune (* application ran from dune exec command *)
-  | Local (* application ran from _build/default folder*)
-  | Debian (* application installed from mina debian package *)
-  | Docker of DockerContext.t
-  | AutoDetect
 (* application ran inside docker container *)
 
-module Executor = struct
-  type t = { official_name : string; dune_name : string; context : context }
```

Comment:
Would be nice if we document the difference between dune_name and official_name here.

Reply:
After seeing the PR train listed [here](https://github.com/MinaProtocol/mina/issues/16712). I'm pretty sure it's unnecessary.